module/Motion/LineTracing.cpp

```diff
@@ -18,7 +18,7 @@ LineTracing::LineTracing(double _targetSpeed, int _targetBrightness, const PidGa
 
 void LineTracing::run()
 {
-  int turnPwm = 0;        // 旋回値を計算
+  double turnPwm = 0;     // 旋回値を計算
```

Comment:
```suggestion
  double turnPwm = 0.0;     // 旋回値を計算
```
影響はないとは思いますが、double 型の Distance 2つと、Straight.h の MIN_PWM に合わせるなら小数点までになります。